crypto/fipsmodule/ml_kem/mlkem/verify.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024-2025 The mlkem-native project authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "verify.h"
+
+#if !defined(MLK_USE_ASM_VALUE_BARRIER) && \
```

Comment:
There is `!` here but not in the comments, which one is intended? I think `if defined`.

Reply:
The code is right and the comment is wrong, I'll fix. 

As a heads-up, for non-compound clauses `foo` the code often contains
```c
#if defined(foo)
...
#else /* foo */
...
#endif /* foo */
```
i.e. we don't distinguish the comments for the positive and negative branches. One could refine this at some point, though I don't think it should take priority. The best way would be to autogenerate those comments in the first place.

Reply:
Those comments are now autogenerated: https://github.com/pq-code-package/mlkem-native/pull/921